src/net/tcp.ts

```diff
@@ -108,13 +108,17 @@ export class Server {
 
   // Invoked when the socket terminates.
   private onDisconnectHandler_ = (info:freedom_TcpSocket.DisconnectInfo) : void => {
-    log.debug('%1: disconnected: %2', this.id_, JSON.stringify(info));
+    if (info) {
```

Comment:
Please add a comment linking to the bug.  I think a TODO to remove this might be appropriate; it seems to me that freedom_TcpSocket's API ought to guarantee that `info` is non-null.